apollo-core/pom.xml

```diff
@@ -31,6 +31,12 @@
 		<github.path>${project.artifactId}</github.path>
 	</properties>
 	<dependencies>
+		<dependency>
```

Comment:
I think it's better to define the cat versions in `dependencyManagement` section of the parent pom.
BTW, how about we use `optional` instead of `provided`? They would achieve the same effect but the semantic is a little different.

```xml
    <dependency>
        <groupId>com.dianping.cat</groupId>
        <artifactId>cat-client</artifactId>
        <optional>true</optional>
    </dependency>
```